kore/src/Kore/AST/Common.hs

```diff
@@ -757,6 +859,12 @@ instance
         <> parameters [forallSort]
         <> arguments' [unparse forallVariable, unparse forallChild]
 
+    unparse2 Forall { forallVariable, forallChild } =
+        "(" <> "∀" <> " "
+        <> (unparse2BindingVariables forallVariable)
+        <> " . " <> (unparse2 forallChild) <> ")"
+
```

Comment:
Please remove all the extra spacing in this module so that there is one blank line between each top-level definition group.